.gitattributes

```diff
@@ -0,0 +1,8 @@
+# Ensure that all files have line feeds for line endings, so that they're consistent regardless of
+# which platform a change was developed on.
+* text=auto eol=lf
+
+# Identify Yarn 2 releases and plugins as binary content.
+# https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored
+/.yarn/releases/** binary
+/.yarn/plugins/** binary
```

Comment:
Interestingly, the version of Yarn (and any plugins) are actually JS files that get committed to the repo. This config tells git to treat them as binary data, so we don't end up with large diffs anytime we update them.